### DISCUSSION TITLE
I can't get EarTube to work on my Windows tablet.

### DISCUSSION THREAD

**Discussion by Jozie-Lore** [community/community#135135] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have been unable to I'll get the EarTube app to work on my laptop... no it's not a laptop it's a tablet. Anyways it does meet all the requirements to run. I'm just not able to figure out how to get it open. Any help would be appreciated! Thank you very much!!

**Comment by LiteBrite82** [comment]
Hi @Jozie-Lore ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
